buster/formatter/base.py

```diff
@@ -0,0 +1,63 @@
+from dataclasses import dataclass
+from typing import Iterable, NamedTuple
+
+# Should be from the `documents` module.
+class Source(NamedTuple):
+    name: str
+    url: str
+    question_similarity: float
+    # TODO Add answer similarity.
+    # answer_similarity: float
```

Comment:
what would the `answer_similarity` be?

Reply:
A score comparing the answer's embedding to the source's embedding, as an extra relevance check.

Reply:
interesting idea!